release-notes/v1_66.md

```diff
@@ -191,6 +191,26 @@ The [Python](https://marketplace.visualstudio.com/items?itemName=ms-python.pytho
 
 ![Interpreters info displayed towards right](images/1_66/active-interpreter-display.png)
 
+### Jupyter
+
+**Kernel Support**
+
+Kernel startup was improved with:
+- Support for more Conda environments.
+- Support for .env files on all platforms.
+
+**Data Viewer**
+
+Named indexes for DataFrames and Series are now supported in the Data Viewer.
```

Comment:
Is it weird to have a blurb about the data viewer but the picture doesn't show the data viewer?